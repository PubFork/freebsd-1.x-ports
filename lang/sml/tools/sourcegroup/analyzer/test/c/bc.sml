functor G (structure D:A) = struct val x = D.y end
