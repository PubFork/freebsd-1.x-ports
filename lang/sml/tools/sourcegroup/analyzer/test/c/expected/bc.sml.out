source sml "c/bc.sml"
  import signature A 
  export functor G ;

